src/style.css

```
@@ -1 +1,7 @@
-/* styles go here */
+.groups > .block:not(:last-child) {
```

won't it work like this?
```suggestion
.block:not(:last-child) {
```